boards/common/blxxxpill/include/periph_conf.h

```diff
@@ -102,13 +102,6 @@ static const timer_conf_t timer_config[] = {
         .bus      = APB1,
         .irqn     = TIM2_IRQn
     },
-    {
-        .dev      = TIM3,
-        .max      = 0x0000ffff,
-        .rcc_mask = RCC_APB1ENR_TIM3EN,
-        .bus      = APB1,
-        .irqn     = TIM3_IRQn
-    },
```

Comment:
No need to remove this definition - TIMER1 and QDEC can't be used together, but they still can be used independent of each other (e.g. an application that doesn't use QDEC could still use all timers)

Reply:
Yes for sure. In that case, should I add all other timers qdec_config[]?

Reply:
I'd leave out the first one as that's used for XTIMER - so `tests/periph_qdec` would break.
But you can add the rest if you want.

Reply:
You are right. Thanks!